cypress/integration/dashboard.spec.ts

```diff
@@ -55,7 +55,7 @@ describe("DashboardUserChoice Test", () => {
 
   it("Should have My Feelings button working", () => {
     cy.contains("My Feelings").click();
-    cy.wait(500);
+    cy.wait(5000);
```

Comment:
it seems like the wait time in #459 was not sufficient I have increased it 10x. I hope it won't fail now